openedx/core/djangoapps/content/course_structures/management/commands/generate_course_structure.py

```diff
@@ -31,16 +30,17 @@ def handle(self, *args, **options):
             course_keys = [CourseKey.from_string(arg) for arg in args]
 
         if not course_keys:
-            logger.fatal('No courses specified.')
+            log.fatal('No courses specified.')
             return
 
-        logger.info('Generating course structures for %d courses.', len(course_keys))
+        log.info('Generating course structures for %d courses.', len(course_keys))
         logging.debug('Generating course structure(s) for the following courses: %s', course_keys)
 
         for course_key in course_keys:
             try:
                 update_course_structure(unicode(course_key))
```

Comment:
I think it would be good to be clear that this is a task that is being run.  So make this an .apply() call.